aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -2371,6 +2371,21 @@ outposts:
   tags:
     product_categories: {}
   version: outposts-2019-12-03
+partner-central:
+  long: '&AWS; &pcsh;'
+  short: '&pcsh;'
+  sort: Partner Central
+  expanded:
+    long: AWS Partner Central
+    short: Partner Central
+  blurb: is a self-service portal where AWS Partners manage their engagement with AWS.
```

Comment:
This is probably fine, but typically the blurb is taken from the landing page, which is: "is a portal for AWS Partners to manage their AWS Partner Network membership."

Reply:
Done